link_rust.sh.in

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# A linker script used when building Rust tests.  Autoconf makes link_rust.sh
+# from link_rust_sh.in, and uses it to pass extra options to the linker
+# when linking Rust stuff.
+#
+# We'd like to remove the need for this, but build.rs doesn't let us pass
+# -static-libasan and -static-libubsan to the linker.
+
+"$CCLD" @RUST_LINKER_OPTIONS@ "$@"
```

Comment:
`$CCLD` needs to be unquoted, see https://travis-ci.org/torproject/tor/jobs/394177687#L2628. we can probably fix travis, but it is quite common to do, for example, `CC='ccache gcc'`.